test/test.js

```diff
@@ -303,13 +303,27 @@ describe('Filter', function() {
     };
 
     beforeEach(function() {
-      f = new F(fixturePath, { persist: true });
+      this.originalCacheRoot = process.env.BROCCOLI_PERSISTENT_FILTER_CACHE_ROOT;
+    });
+
+    afterEach(function() {
+      process.env.BROCCOLI_PERSISTENT_FILTER_CACHE_ROOT = this.originalCacheRoot || '';
```

Comment:
I think we want it to be unset entirely if it wasn't set originally, not just empty. So wouldn't it need a delete?